static/src/javascripts/projects/common/modules/commercial/hosted-video.js

```diff
@@ -19,8 +19,7 @@ define([
     videojs,
     loadingTmpl
 ) {
-
-    var $videoEl = $('.vjs-hosted__video');
+    var $videoEl;
```

Comment:
If it's just being used in `init`, though, perhaps it makes sense to keep the var declaration scoped there? Just to keep the outer scope as simple as possible.